src/mono/sample/wasm/wasm.mk

```diff
@@ -35,7 +35,7 @@ run-browser:
 		echo "The tool dotnet-serve could not be found. Install with: $(DOTNET) tool install --global dotnet-serve"; \
 		exit 1; \
 	else  \
-		$(DOTNET) serve -d:bin/$(CONFIG)/AppBundle $(OPEN_BROWSER) -p:8000; \
+		$(DOTNET) serve -d:bin/$(CONFIG)/AppBundle -h "Cross-Origin-Opener-Policy:same-origin" -h "Cross-Origin-Embedder-Policy:require-corp" $(OPEN_BROWSER) -p:8000; \
```

Comment:
since this is fairly magical it might be good to have a comment here explaining why we're setting these headers (and doing it in wasm.mk), i.e. (i don't know the syntax for comments in mk files)

```
# we set specific headers to enable SharedArrayBuffer support in browsers for threading: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer#security_requirements
```